src/vis/pcp/PCPVis.tsx

```diff
@@ -65,7 +76,7 @@ export function PCPVis({ config, extensions, columns, setConfig, hideSidebar = f
   }, [traces]);
 
   return (
-    <div className="d-flex flex-row w-100 h-100" style={{ minHeight: '0px' }}>
+    <div ref={plotlyDivRef} className="d-flex flex-row w-100 h-100" style={{ minHeight: '0px' }}>
       <div
         className={`position-relative d-flex justify-content-center align-items-center flex-grow-1 ${
```

Comment:
This fixes a bug in the Vis. This should probably be fixed in the develop branch to avoid branches diverging